Tamil

TamilNeram opened the pull request:
## 📌 Description
Updated 2 missing entries in Tamil Language file

## 🎨 Visual Changes (Screenshots / Videos)
If applicable, attach relevant screenshots or screen recordings showcasing the modifications.

## 🔗 Related Issues
- Closes #<issue_number> (if this PR fixes an issue)
- Related to #<issue_number> (if applicable)

## ✅ Checklist
- ✅  I have read and followed the [Contributing Guidelines](https://github.com/XengShi/materialYouNewTab/blob/main/CONTRIBUTING.md).
- [ ] I have tested my changes thoroughly to ensure expected behavior.
- [ ] I have verified compatibility across Chrome and Firefox (additional browsers if applicable).
- [ ] I have attached relevant visual evidence (screenshots/videos) if applicable.
- ✅  My code follows the project's coding style and conventions.
- [ ] I have updated the [CHANGELOG.md](https://github.com/XengShi/materialYouNewTab/blob/main/CHANGELOG.md) under the appropriate categories with all my changes in this PR.

prem-k-r commented:
Reflecting old changes as well, can you please rebase you last commit on main branch

prem-k-r commented:
Consider re-removing comments that were meant for instructions, for example `// இதைச் சிறியதாக வைத்திரு`

TamilNeram commented:
Rebased and updated, all the remaining files are in sync. Only the Tamil Translation file got changed. 

I want to keep the comments also in Tamil.  Hope this is fine with you.

prem-k-r commented:
Nice, thanks! 

https://github.com/XengShi/materialYouNewTab/discussions/195#discussioncomment-13411767
If you like, you can also check this out.

TamilNeram commented:
_Merging is blocked

    At least 1 approving review is required by reviewers with write access.
    Cannot update this protected ref._

What is this check? How to fix this?

prem-k-r commented on locales/ta.js:
```diff
@@ -77,7 +77,7 @@ const ta = {
 
     // Weather
-    "WeatherApiText": "உங்கள் WeatherAPI விசையை உள்ளிடவும்",
+    "WeatherApiText": "உங்கள் WeatherAPI விசையை உள்ளிடவும்",    // 'வானிலைபநிஇ' சரத்திற்கு, ஒலிபெயர்ப்பு மட்டும்
```
Only one last comment left :)

prem-k-r commented:
Some of the brand name seems different:
![image](https://github.com/user-attachments/assets/c6e55ad3-c3e5-49ab-ad7b-a51421257a7c)
![image](https://github.com/user-attachments/assets/717e7942-c8ad-4b07-b5be-11b6c0f009d9)
![image](https://github.com/user-attachments/assets/eab1bafd-32b9-4f88-8119-b7a9f2174c00)

XengShi commented:
Some words should be transliterated instead of translated, like "brave" in "braveEngine," which should be "ப்ரேவ்," while "துணிச்சல்" means "courage" but isn't the right transliteration; I hope this clears up any confusion for users.

prem-k-r reviewed (changes requested):
Please transliterate instead of direct translations for the brand names

TamilNeram commented:
Can we have 2 versions in Tamil. like Tamil(ta-IN) and Tamil(ta). 

ta-IN to have the translitration. 

ta- Which I would like to see and use.
